packages/www/docs/api-reference/vod/mint-nft.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: Mint a Video NFT
+description: Livepeer.com API Reference | Mint
+metaTitle: Mint - Livepeer.com
+metaDescription: Livepeer.com API Reference | Mint
+---
+
+# How do I use Livepeer to mint a video NFT?
+
+You can use
+[this script](https://gist.github.com/victorges/3369cc04c0002a63b2fab33079d8827e)
+(`bash`) to have a local file uploaded to Livepeer and then export it to IPFS.
```

Comment:
We should update this to use the `@livepeer/video-nft` script instead. I think we can merge this is as is though, and I can make a follow-up PR with the updated script and all